.github/workflows/helm-chart-ci.yaml

```diff
@@ -33,6 +33,31 @@ jobs:
       - name: Verify Docs updated
         run: ./helm-docs.sh
 
+  ensure-image-overrides:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.4.0
+      - name: Check objects for images without overrides
+        run: |
+          set +e
+          set -o pipefail
+          # Look for image: defintions that are not templated. If we find none, exit is not 0 and we invert the error code to get the
+          # test to pass
+          grep -r "image:" "charts/spire"  | grep "templates/" | grep -v 'image: {{ template "' > /tmp/findings
```

Comment:
Do you need the quotes around `charts/spire`?

Reply:
```suggestion
          grep -r "image:" charts/spire | grep "templates/" | grep -v 'image: {{ template "' > /tmp/findings
```

Reply:
No, but there isn't really a need to put explicit quotes around the image: either. so I was being consistent.